host/src/pack/dpkg/opt/cfc/mwc/bin/lg_launcher.sh

```diff
@@ -62,6 +62,14 @@ fi
 
 $FOLDER_1_UPDATE
 $FOLDER_2_UPDATE
+
```

Comment:
need change civ_startapp.lock to civ_startapp_$THEUSER.lock to fix multi-user issue